qiita_pet/templates/workflows.html

```diff
@@ -63,16 +64,26 @@
     <h3>Recommended Workflows</h3>
     ~~ Click on the spheres to get more information ~~
```

Comment:
What do you think about this:

<img width="473" alt="Screen Shot 2021-03-16 at 4 56 47 PM" src="https://user-images.githubusercontent.com/375307/111394776-9e99a900-8678-11eb-8363-4f4366d18370.png">


```suggestion
    <h5>Click on the circles to get more information</h5>
```